src/components/widget/SelectionDropdown.js

```diff
@@ -301,15 +301,14 @@ export default class SelectionDropdown extends Component {
   renderEmpty = () => this.renderHeader(this.props.empty);
 
   loading = this.renderHeader(
-    <ReactCSSTransitionGroup
-      transitionName="rotate"
-      transitionEnterTimeout={1000}
-      transitionLeaveTimeout={1000}
+    <CSSTransition
+      className="rotate icon-rotate"
```

Comment:
Might be wrong in here but shouldn't this be classNames=""  instead of className="" (https://reactcommunity.org/react-transition-group/css-transition)

Reply:
Weird, cause when I tried their repl it was throwing an error, and in our case everything works fine as far as I can tell, and nothing gets animated when I use `classNames`.